chain-vote/src/gargamel.rs

```diff
@@ -67,16 +67,16 @@ impl Ciphertext {
         }
     }
 
-    pub fn to_bytes(&self) -> Vec<u8> {
-        let mut r = self.e1.to_bytes().to_vec();
-        r.extend_from_slice(&self.e2.to_bytes());
+    pub fn to_bytes(&self) -> [u8; 130] {
```

Comment:
This may be too much to return on stack. I'd rather kept it a `Vec`. This also gets rid of a magic size in the API.